telegraf/1.21/alpine/Dockerfile

```diff
@@ -23,8 +29,7 @@ RUN set -ex && \
     mv /usr/src/telegraf*/etc/telegraf/telegraf.conf /etc/telegraf/ && \
     mkdir /etc/telegraf/telegraf.d && \
     cp -a /usr/src/telegraf*/usr/bin/telegraf /usr/bin/ && \
-    gpgconf --kill all && \
-    rm -rf *.tar.gz* /usr/src /root/.gnupg && \
+    rm -rf *.tar.gz* /usr/src && \
```

Comment:
what about `/root/.gnupg`?

Reply:
It should be good! https://github.com/influxdata/influxdata-docker/pull/643#discussion_r961762946